src/app/project/documents/documents-tab.component.ts

```diff
@@ -94,260 +80,303 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
+        this.queryParams['documentType'] ||
+        this.queryParams['datePostedEnd']
+      ) {
+        this.showAdvancedFilters = true;
       }
+
+      this.loadingTableParams = false;
+      this._changeDetectionRef.detectChanges();
     });
 
-    this.searchService.getSearchResults(
-      '',
-      'Document',
-      [
-        { name: 'project', value: this.currentProject._id },
-        { name: 'categorized', value: false }
-      ],
-      this.storageService.state[this.currentProject._id].tableParams ? this.storageService.state[this.currentProject._id].tableParams.currentPage : 1,
-      this.storageService.state[this.currentProject._id].tableParams ? this.storageService.state[this.currentProject._id].tableParams.pageSize : 10,
-      this.storageService.state[this.currentProject._id].tableParams ? this.storageService.state[this.currentProject._id].tableParams.sortBy : '-datePosted',
-      { documentSource: 'PROJECT' },
-      true,
-      null,
-      this.filterForAPI,
-      ''
-    )
-    .takeUntil(this.ngUnsubscribe)
-    .subscribe((res: any) => {
-      if (res[0].data.meta && res[0].data.meta.length > 0) {
-        this.hasUncategorizedDocs = true;
-        this.loading = false;
-        this._changeDetectionRef.detectChanges();
+    this.documentService.getValue().pipe(takeWhile(() => this.alive)).subscribe((searchResults: SearchResults) => {
+      this.tableData.totalListItems = searchResults.totalSearchCount;
+      this.tableData.items = searchResults.data.map(record => {
+        record['showFeatured'] = true;
+        return { rowData: record };
+      });
+
+      if (this.tableData.totalListItems > 10) {
+        this.tableData.options.showPageSizePicker = true;
+      } else {
+        this.tableData.options.showPageSizePicker = false;
       }
+
+      this.tableData.columns = this.tableColumns;
+      this.loadingtableData = false;
+
+      this._changeDetectionRef.detectChanges();
     });
   }
 
+  private setFilters() {
+    const docDateFilter = new FilterObject(
+      'issuedDate',
+      FilterType.DateRange,
+      '', // if you include a name, it will add a label to the date range filter.
+      new DateFilterDefinition('datePostedStart', 'Start Date', 'datePostedEnd', 'End Date'),
+      6
+    );
+
+    const milestoneFilter = new FilterObject(
+      'milestone',
+      FilterType.MultiSelect,
+      'Milestone',
+      new MultiSelectDefinition(
+        this.milestoneArray,
+        [],
+        this.legislationFilterGroup,
+        null,
+        true
+      ),
+      6
+    );
+
+    const documentAuthorTypeFilter = new FilterObject(
+      'documentAuthorType',
+      FilterType.MultiSelect,
+      'Document Author',
+      new MultiSelectDefinition(
+        this.documentAuthorTypeArray,
+        [],
+        this.legislationFilterGroup,
+        null,
+        true
+      ),
+      4
+    );
+
+    const documentTypeFilter = new FilterObject(
+      'documentType',
```

Comment:
The doc type filter isn't working at the moment.  I think this needs to be `type` instead of `documentType`? (Based on my observation of the test environment)

Reply:
ee2bedf